GroupMeClientApi/GroupMeClient.cs

```diff
@@ -22,6 +23,11 @@ public class GroupMeClient
         /// </summary>
         internal const string GroupMeAPIUrlV2 = "https://v2.groupme.com/";
 
+        /// <summary>
+        /// The Base URL for GroupMe's V4 API.
+        /// </summary>
+        internal const string GroupMeAPIUrlV4 = "https://api.groupme.com/v4";
+
```

Comment:
I think these can be consolidated some later. ```GroupMeApiUrlV4``` only shows up here, so it can be private and not internal, right?